src/babashka/process.clj

```diff
@@ -11,18 +30,21 @@
                       :in :in-enc
                       :out :out-enc
                       :dir
+                      :env
                       :timeout
                       :throw
                       :wait]
                :or {out :string
                     err :string
                     dir (System/getProperty "user.dir")
+                    env ::jvm-env
```

Comment:
I think we can leave this weird keyword default out. If env is nil, we shouldn't do anything, so the condition below can be `(map? env)`.

E.g.:

```
$ FOO=1 clj
Clojure 1.10.2-alpha1
user=> (System/getenv "FOO")
"1"
user=> (require '[clojure.java.shell :refer [sh]])
nil
user=> (sh "bash" "-c" "echo $FOO")
{:exit 0, :out "1\n", :err ""}
user=> (sh "bash" "-c" "echo $FOO" :env {})
{:exit 0, :out "\n", :err ""}
user=> (sh "bash" "-c" "echo $FOO" :env nil)
{:exit 0, :out "1\n", :err ""}
```

Reply:
Agreed. It's a hold-over from an earlier version and is no longer needed. Appendectomy complete.

Reply:
Doesn't that already work, since `as-string-map` passes through the object if it's not a Clojure map? Btw, I think the `when-let` in `set-env` can go as well, since we already checked if it's truthy earlier.

Reply:
Yup: Addressed here https://github.com/babashka/babashka.process/pull/11